_schemaV2.graphql

```diff
@@ -15196,6 +15227,12 @@ type Viewer {
   previewSavedSearch(
     attributes: PreviewSavedSearchAttributes
   ): PreviewSavedSearch
+  recentlySoldArtworks(
```

Comment:
**nitpick:** Adding a description to this connection would improve the formatting of this file.

**question:** Why does this appear twice in the schema?

Reply:
> Why does this appear twice in the schema?

First one it's a field in the `Query` type the second one in the `Viewer` type.
Here is a bit described the `Viewer` type https://artsy.slack.com/archives/CP9P4KR35/p1599075665072400